soundmanager.h

```diff
@@ -8,7 +8,14 @@ class SoundSample
     Mix_Chunk* chunk;
     int samplefreq, bytespersample; //TODO: this does not quite belong here, I think
 public:
+    /** ctor */
     SoundSample(const char *filename, int samplefreq, int bytespersample);
+    
+    /** copy ctor is deleted */
```

Comment:
Also noise, this literally states what's on the next line.

On the other hand, it might be meaningful to explain _why_ the copy constructor is deleted, which I assume to be because it's a hassle to make sure all wrapped resources are properly managed so it's safer to just disallow copying.